.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,53 @@
+// If you want to run as a non-root user in the container, see .devcontainer/docker-compose.yml.
+{
+  "name": "ac-dev-server",
+
+  // Update the 'dockerComposeFile' list if you have more compose files or use different names.
+  // The .devcontainer/docker-compose.yml file contains any overrides you need/want to make.
+  "dockerComposeFile": [
+    "../docker-compose.yml",
+    "../docker-compose.override.yml", // needed until this issue will be solved: https://github.com/microsoft/vscode-remote-release/issues/1080
```

Comment:
Got the following when trying to start the devcontainer:

```
[2021-04-21T05:02:05.779Z] [PID 63629] [3 ms] Remote-Containers 0.166.1 in VS Code 1.55.2 (3c4e3df9e89829dce27b7b5c24508306b151f30d).
[2021-04-21T05:02:05.912Z] [PID 63629] [136 ms] Start: Resolving Remote
[2021-04-21T05:02:05.913Z] [PID 63629] [137 ms] Setting up container for folder or workspace: /Users/ple/Documents/GitHub/azerothcore-wotlk
[2021-04-21T05:02:05.916Z] [PID 63629] [140 ms] Start: Check Docker is running
[2021-04-21T05:02:05.916Z] [PID 63629] [140 ms] Start: Run: docker version --format {{.Server.APIVersion}}
[2021-04-21T05:02:06.093Z] [PID 63629] [316 ms] Server API version: 1.41
[2021-04-21T05:02:06.117Z] [PID 63629] [341 ms] Start: Run: docker-compose version --short
[2021-04-21T05:02:06.551Z] [PID 63629] [775 ms] Start: Run: docker ps -q -a --filter label=com.docker.compose.project=azerothcore-wotlk --filter label=com.docker.compose.service=ac-dev-server
[2021-04-21T05:02:06.704Z] [PID 63629] [928 ms] Start: Run: docker-compose -f /Users/ple/Documents/GitHub/azerothcore-wotlk/docker-compose.yml -f /Users/ple/Documents/GitHub/azerothcore-wotlk/docker-compose.override.yml -f /Users/ple/Documents/GitHub/azerothcore-wotlk/.devcontainer/docker-compose.yml config --services
[2021-04-21T05:02:07.113Z] [PID 63629] [1337 ms] 
[2021-04-21T05:02:07.113Z] [PID 63629] [1337 ms] .FileNotFoundError: [Errno 2] No such file or directory: '/Users/ple/Documents/GitHub/azerothcore-wotlk/docker-compose.override.yml'

[2021-04-21T05:02:07.113Z] [PID 63629] [1337 ms] Exit code 1
[2021-04-21T05:02:07.116Z] [PID 63629] [1340 ms] Command failed: docker-compose -f /Users/ple/Documents/GitHub/azerothcore-wotlk/docker-compose.yml -f /Users/ple/Documents/GitHub/azerothcore-wotlk/docker-compose.override.yml -f /Users/ple/Documents/GitHub/azerothcore-wotlk/.devcontainer/docker-compose.yml config --services
```

Reply:
This is normal, to use the dev-container you should copy the docker-compose.override.yml provided in the /conf/dist/ folder.

That's an empty file that you can even extend with personal confs.

I forgot to add this step to the documentation. I'm gonna do it now, thanks!

Reply:
I've added the instructions to this post and the wiki PR as well. Please, let me know if you can open the dev-container and also use the debugging feature

Reply:
gotcha, will test that out.

Reply:
now i am getting this:
```
[2021-04-21T06:49:59.433Z] [PID 36668] Successfully built 594c7451b04e846fb90298dc76026980fb63411ce356136a1eed8ddb47d62757
[2021-04-21T06:49:59.490Z] [PID 36668] azerothcore-wotlk_ac-database_1 is up-to-date
[2021-04-21T06:49:59.502Z] [PID 36668] 
ERROR: for ac-dev-server  Container "641644a1498c" is unhealthy.
[2021-04-21T06:49:59.503Z] [PID 36668] ERROR: Encountered errors while bringing up the project.
[2021-04-21T06:49:59.677Z] [PID 36668] [168943 ms] Command failed: docker-compose --project-name azerothcore-wotlk -f /Users/ple/Documents/GitHub/azerothcore-wotlk/docker-compose.yml -f /Users/ple/Documents/GitHub/azerothcore-wotlk/docker-compose.override.yml -f /Users/ple/Documents/GitHub/azerothcore-wotlk/.devcontainer/docker-compose.yml up -d --build ac-dev-server ac-database
```

Reply:
ahh, looks like i had a conflict from a previous test. doing `docker system prune -a` and `docker system prune --volumes` resolved it.